### ISSUE TITLE
SyncLab: Pictures have buggy fill

### ISSUE THREAD

**Issue by alexfjw** [PowerPointLabs/PowerPointLabs#1612]
As above, pictures seem to have a fill that's somewhat messed up like textboxes. (Fill changes after copying in synclab)
To be verified and fixed after #1607 is merged, either as a new PR or into #1594

**Comment by damithc** [comment]
From what I can remebmer, there is a legit use for `fill` for pictures. e.g., when the image has a transparent background.

**Comment by alexfjw** [comment]
Thanks, I'll check that out. Code wise, shapes also to have a different setting for the color of transparent backgrounds, it might be related though.

**Comment by alexfjw** [comment]
Closing, works fine on ppt2016 with #1067 merged. Looks like #1607 fixes this issue as well.
